partials/index/Standard.njk

```diff
@@ -0,0 +1,179 @@
+
+{% macro standard(asyncapi) %}
+import { 
+  Client, 
+  NatsConnectionOptions, 
+  connect,
+  Payload, 
+  NatsError, 
+  Subscription, 
+  ServersChangedEvent, 
+  SubEvent, 
+  ServerInfo
+  } from 'ts-nats';
```

Comment:
the package.json needs to have a way of enforcing the client that you are expecting. Otherwise any kind of change such as the upcoming v2 will break your generator.

Reply:
@aricart can you elaborate on the release date for v2? And can you imagine people would like to use v1 rather then v2 in any scenario in regards to this template?